modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.scss

```diff
@@ -0,0 +1,7 @@
+.management-bar-separator-left::before {
+	background-color: #6b6c7e;
```

Comment:
We should be using a sass variable here :). Also, it seems that this color is not the same as in the design, right? I think it may be [Secondary L2](https://clayui.com/docs/css/color.html) $gray-400 in sass

Reply:
Thanks @victorg1991! ❤️  After some digging [I finally found it in figma](https://www.figma.com/file/DR86VmK5OPyxuljwn7ujPP/lexi-1254-fix-info-button-to-the-right?node-id=21%3A25701), looks like it's Secondary L1. On it!

Reply:
Good find! I didn't find the figma issue, just by comparing 😂  It seems that my eyes are not well trained

Reply:
To be fair you were closer than me 😅